DRM schema versioning

aucuparia opened the pull request.

Commit:
Fix double / in coll_path

Doesn't seem to be causing any issues (at the moment), but cleaner this way

lwesterhof commented on datarequest.py:
```diff
@@ -554,22 +556,30 @@ def datarequest_reviewers_get(ctx, request_id, pending=False):
 
     :returns: Dict with schema and UI schema
     """
+    log.write(ctx, '=======================================')
```
Debug statements can be removed?

aucuparia replied:
Done.

lwesterhof commented on datarequest.py:
```diff
@@ -66,7 +66,9 @@
-SCHEMACOLLECTION  = constants.UUSYSTEMCOLLECTION + "/datarequest/schemas/youth-0"
+SCHEMACOLLECTION  = constants.UUSYSTEMCOLLECTION + "/datarequest/schemas"
+SCHEMA_URI_PREFIX = "https://yoda.uu.nl/datarequest/schemas/"
+SCHEMA_VERSION    = "youth-1"
```
In a later iteration we can make this a configuration option, so that the schema is not hard coded.

aucuparia replied:
Added to backlog.

aucuparia committed:
Add version parameter to datarequest_schema_get

aucuparia committed:
Add schema ID to datarequest on submission

aucuparia committed:
Add $id field to youth-1 datarequest schema

aucuparia committed:
Lint fixes

aucuparia reviewed (commented):
All issues solved.

lwesterhof committed:
Fix flake8 issues.

lwesterhof reviewed (approved):
Because one of the schemas is a symlink we get an error, because the two schemas have the same name.

Don't the other schemas like assignment and datamanager review need a schema id as well?

aucuparia committed:
Don't symlink datarequest schema

This interferes with JSON schema validation in Github CI because of the duplicate schema ID.

aucuparia committed:
Add schema ID to all datarequest schemas

aucuparia commented:
> Because one of the schemas is a symlink we get an error, because the two schemas have the same name.
> 
> Don't the other schemas like assignment and datamanager review need a schema id as well?

Symlink is gone now, and I've added $id fields to the other schemas as well.

lwesterhof reviewed (approved):
LGTM
